src/Navbar.css

```diff
@@ -0,0 +1,52 @@
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 3em;
+  color: white;
+  background-color: #5757ff;
+}
+
+.col-1 {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 1rem;
+}
+
+.btn {
```

Comment:
choose unique classes name, because if we will add bootstrap in the future so the classes will be conflicted